### ISSUE TITLE
🐛 `git show $revision:path/to/file.ext`: lose syntax highlighting after "#" lines

### ISSUE THREAD

**Issue by matheustavares** [dandavison/delta#1163]
Delta seems to lose syntax highlighting when running `git show $revision:path/to/file.ext` on a file that contains a line starting with "`#`":

![delt](https://user-images.githubusercontent.com/12701583/185263903-0c365ab2-fa25-4e5c-af0c-776ecb42f270.png)

Output without pager:

```bash
$ git --no-pager show :file.c
int zero(void)
{
        return 0;
}

#include "another_file.h"

int main()
{
        return zero();
}
```

**Comment by dandavison** [comment]
Hi @matheustavares, this is actually fixed on master by #1127, but I've been a bit slack about making a release. Sorry about that -- if you're able to [build from source](https://dandavison.github.io/delta/build-delta-from-source.html) then great, otherwise I will release soon.

**Comment by matheustavares** [comment]
Thanks @dandavison ! Yes, I'll build from source :)
